tests/TransactionBuilderTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\DB;
+
+class TransactionBuilderTest extends TestCase
+{
+    protected $insertData = ['name' => 'klinson', 'age' => 20, 'title' => 'admin'];
+    protected $originData = ['name' => 'users', 'age' => 20, 'title' => 'user'];
```

Comment:
It seems odd to me that the initial data for the collection is expressed as a single document. Is there a benefit to seeding the collection under test with multiple documents?

I'll note that some of the test methods below do refer to this as a single doc (`testUpdate()` modifies it), so that would need to change if so.

Reply:
Done